doc/source/user_guide/ug_running.rst

```diff
@@ -837,6 +834,47 @@ modify the scripts and input settings in the case directory, NOT the run directo
 In general, files in the run directory are overwritten by versions in the case
 directory when the model is built, submitted, and run.
 
+.. _aliases:
+
+Useful aliases
+-------------------
+
+This section provides a list of some potentially useful aliases that leverage the CICE 
+scripts.  These are not defined by CICE and are not required for using CICE.  They
+are provided as a sample of the types of things that can be done by users.
+
+The current **ice_in**, **cice.settings**, and **env.[machine]** files are copied from 
+the case directory into the run directory when the model is run.  Users can create aliases 
+leveraging the variables in these files.  Aliases like the following can be established 
```

Comment:
Everyone has their own style to usage in their home shell.  I prefer to avoid telling people how to work.  My thought here is that we should (at most) demonstrate some possible aliases without encouraging them or micro managing how people work.

Reply:
I understand that, but some users (I'm thinking students) might not know at all how they should use these aliases. By mentioning "shell startup files", it at least gives them something they can Google to learn how to do it. More advanced users that want to work differently can just disregard that sentence and do as they please.

Reply:
I'm updated this section a bit and I think it's clearer.  Thanks for the feedback.